vm/actor/src/util/mod.rs

```diff
@@ -9,9 +9,10 @@ pub mod puppet;
 mod set;
 mod set_multimap;
 pub mod smooth;
-pub mod unmarshallable;
+mod unmarshallable;
 
 pub use self::balance_table::BalanceTable;
 pub use self::multimap::*;
 pub use self::set::Set;
 pub use self::set_multimap::SetMultimap;
+pub use unmarshallable::*;
```

Comment:
no I didn't mean making the type public, this type should not be exposed at all

Reply:
Its used in the puppet_actor_test for the serialization test i believe

Reply:
ah, right. This test probably shouldn't live there. The `tests` directory should just be used to test publicly exposed functions. Something like that could just live as a test module within the code (around where the Unmarshallable struct is defined).

Probably shouldn't expose a type like that just so that it can be tested somewhere else